task/sbom-json-check/0.1/README.md

```diff
@@ -0,0 +1,22 @@
+# sbom-json-check task (Skipped)
+
+## Description:
+
+`sbom-json-check` task verifies the integrity and security of a [Software Bill of Materials](https://www.cisa.gov/sbom) (SBOM) file in JSON format. This task checks the syntax of the sbom-cyclonedx.json file (found in the `/root/buildinfo/content_manifests/` directory) using the [CyloneDX](https://cyclonedx.org/) tool. This tool is being led by longtime security community leader Open Web Application Security Project ([OWASP](https://owasp.org/)). CycloneDX is a self-defined “lightweight SBOM standard designed for use in application security contexts and supply chain component analysis.”
+
+In general, an SBOM is a comprehensive list of all the software components and software dependencies that are involved in the development and delivery of an application, along with information such as version numbers and licensing information. [Refer to this article](https://www.redhat.com/en/blog/how-red-hat-addressing-demand-develop-offerings-more-securely) to learn more about SBOM.
+
+This task is skipped.
```

Comment:
This task is not skipped, it's a required check.

Reply:
Thank you for pointing this out, I forgot to update it here earlier.
Updated now.

Reply:
Thanks!